src/common/libkvs/treeobj.c

```diff
@@ -263,6 +263,84 @@ int treeobj_insert_entry (json_t *obj, const char *name, json_t *obj2)
     return 0;
 }
 
+json_t *treeobj_copy (json_t *obj)
```

Comment:
would `json_deep_copy()` work here?

Reply:
naming wise?  That would follow jansson style naming.  Are you thinking ```treeobj_copy()``` would be the shallow equivalent?

Reply:
I meant could the jansson function `json_deep_copy()` do what `treeobj_copy()` proposes to do here?  

http://jansson.readthedocs.io/en/2.8/apiref.html#copying

Reply:
ohhh, duh!  You're right.  I'm going to rename to ```treeobj_deep_copy()``` as well, as that would follow the jansson style.  ```treeobj_copy()``` will be the treeobj equivalent of shallow copy.